libsolidity/codegen/CompilerUtils.h

```diff
@@ -66,7 +66,7 @@ class CompilerUtils
 	void revertWithStringData(Type const& _argumentType);
 
 	/// Computes the absolute calldata offset of a tail given a base reference and the (absolute)
-	/// offset of the tail pointer. Performs bounds checks. For dynamically sized types it also
+	/// offset of the tail pointer. Performs bounds checks. If @a _type is a dynamically sized type it also
```

Comment:
I would really like to say "array" here to avoid confusion with "dynamically encoded type".